seqr/views/utils/dataset_utils.py

```diff
@@ -385,9 +371,18 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
     if missing_cols:
         raise ValueError(f'Invalid file: missing column(s) {", ".join(sorted(missing_cols))}')
 
+    sample_id_to_tissue_type = {}
     for line in tqdm(f, unit=' rows'):
         row = dict(zip(header, _parse_tsv_row(line)))
-        for sample_id, row_dict in parse_row(row, sample_id_to_tissue_type=sample_id_to_tissue_type):
+        for sample_id, row_dict in parse_row(row):
+            tissue_type = TISSUE_TYPE_MAP.get(row_dict.pop(TISSUE_COL, None), None)
+            if sample_id in sample_id_to_tissue_type:
+                prev_tissue_type = sample_id_to_tissue_type[sample_id]
+                if tissue_type != prev_tissue_type:
+                    raise ValueError(f'Mismatched tissue types for sample {sample_id}:'
+                                     f' {REVERSE_TISSUE_TYPE[prev_tissue_type]}, {REVERSE_TISSUE_TYPE[tissue_type]}')
```

Comment:
We allow only one tissue type per sample in each input data file.

Reply:
if so, then theres no need to keep a dictionary of a sample to tissue type mapping. It would be much neater to keep a single `tissue_type` variable, validate all the samples in the file have that tissue type, and then pass that tissue type directly

Reply:
I thought that different samples could have different tissue types. The example data file that Ben provided has a single tissue type. Maybe we should conform with Ben if it is always the case.

Reply:
Your comment here is "We allow only one tissue type per sample in each input data file." You are currently failing the file if any sample has a different tissue type. So either you should clean up the code if this is the case, or you need to completely change this validation

Reply:
I realize I misread this - we are allowing one tissue type per sample, not one tissue type per file